build/slug.d.ts

```diff
@@ -0,0 +1 @@
+export declare function generateRandomSlugs(amount: number): string[];
```

Comment:
The folder "build" shouldn't be committed to the Repo.

Reply:
I try to but build in gitIgnore but I seems like it does not work

Reply:
Tried to add to file .gitignore at the root of the project?

Reply:
Another tip... not that you committed those files, you have to delete them, commit, and only after add to .gitignore

Reply:
I made a new commit without all the ignore files, do I need to make a new PR for this? or first another commit with the files but in the gitignore files mentioned, and then both commits in one PR?

Reply:
No. Just by pushing to the same branch will already update de PR contents